src/main/java/seedu/address/logic/commands/BoltCommand.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_BOLT;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Bolt;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a bolt to a student in the address book.
+ * @code bolt 1 b/3
+ */
+public class BoltCommand extends Command {
+
+    public static final String COMMAND_WORD = "bolt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds the number of lightning bolts to "
+            + "the person identified by the index number used in the last person listing. "
+            + "The number of lightning bolts corresponds to the number of absences. "
+            + "Updates the existing absence count by adding the input number.\n"
+            + "Parameters: INDEX (must be a positive integer) "
```

Comment:
in the `Bolt` class, it accepts a value of 0? Should we change this to be "must be a nonnegative integer"?